src/components/PeopleAndRoles/ListPeopleAndRoles.vue

```diff
@@ -42,11 +42,11 @@
           </v-tooltip>
 
           <br v-if="orgPerson.action">
-          <v-chip v-if="orgPerson.action === ActionTypes.ADDED"
+          <v-chip v-if="wasAdded(orgPerson)"
             x-small label color="#1669BB" text-color="white">ADDED</v-chip>
-          <v-chip v-if="orgPerson.action === ActionTypes.EDITED"
+          <v-chip v-if="wasEdited(orgPerson)"
             x-small label color="#1669BB" text-color="white">CORRECTED</v-chip>
-          <v-chip v-if="orgPerson.action === ActionTypes.REMOVED"
+          <v-chip v-if="wasRemoved(orgPerson)"
```

Comment:
I made this little CHIP section into a small common component as i believe we will be applying the CHIP in nearly every corrected component. No impact here, just an fyi.

Reply:
Thanks.
If it's worth it, we can update the existing v-chips later.